fi/development.html

```diff
@@ -0,0 +1,210 @@
+    <link rel="icon" href="/static/SymPy-Favicon.ico" />
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>
+    <script type="text/javascript" src="/static/javascript/main.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1><a href="index.html"> <img src="/static/images/logo.png" alt="SymPy Logo" /> SymPy</a></h1>
+        <div id="mobile-menu"><i class="icon-ellipsis-vertical"></i></div>
+      </header>
+      <nav id="main-navigation" class="row">
+        
+        
+        <ul>
+          <li class=""><a href="index.html">Pääsivu</a></li>
+          <li class=""><a href="features.html">Ominaisuudet</a></li>
+          <li class=""><a href="download.html">Lataa</a></li>
+          <li class=""><a href="http://docs.sympy.org">Dokumentaatio <i class="icon-external-link-sign"></i></a></li>
+          <li class=""><a href="support.html">Tuki</a></li>
+          <li class="active"><a href="development.html">Kehitys</a></li>
+          <li class=""><a href="donate.html">Lahjoita</a></li>
+          <li><a href="http://live.sympy.org/">Online-komentokehoite <i class="icon-external-link-sign"></i></a></li>
+        </ul>
+        
+      </nav>
+      <section class="row" id="main">
+        <div class="col span_7 clr small_span_4">
+          
+          <article id="content">
+            
+
+<h2 class="title">Kehitys</h2>
+<div class="inner">
+  <p class="first">
+    Me olemme kiinnostuneita mistä tahansa avustuksesta. On olemassa monta tapaa jolla voit
+      avustaa:
+    <ul>
+      <li>
+        Mikäli kirjoitit jotakin kiinnostavaa SymPy-ohjelmistolla, ole hyvä ja lähetä meille korjaustiedost
+         lähdekoodilla (ole hyvä ja katso lisätiedot alla).
+      </li>
+
+      <li>
+        Mikäli havaitsit bugin (tai haluat meidän tietävän mitä ajattelet), kerro meille
+         <a href="http://groups.google.com/group/sympy">postituslistalla</a>. Voit myös
+        lisätä bugiraportin (tai kommentin) tällä <a href="https://github.com/sympy/sympy/issues/">sivulla</a>.
+      </li>
+      <li>Voit blogata kuinka käytät SymPy-ohjelmistoa.</li>
+      <li>Voit avustaa meitä kirjoittamaan dokumentaatiota (lähettämällä meille korjaustiedoston).</li>
+      <li>Voit avustaa ihmisiä <a href="http://groups.google.com/group/sympy">postituslistalla</a>.</li>
+      <li>Voit avustaa meitä tarkistamalla korjaustiedostoja.</li>
+      <li>Voit lähettää meille korjaustiedostoja joissa on bugikorjauksia tai uusia ominaisuuksia.</li>
+    </ul>
+  </p>
+  <p>
+    Paras tapa avustaa lahjoittamalla lähdekoodia tai dokumentaatiota on lähettämällä meille korjaustiedoston.
+  </p>
+</div>
+
+<h2 class="title">Kuinka lähettää korjaustiedosto</h2>
+<div class="inner">
+  <p class="first">
+    Paras tapa on lähettää meille pull request
+    <a href="http://github.com/sympy/sympy">sympy/sympy</a> repositoryyn.
```

Comment:
I think we should try to find translations for 'pull request' and 'repository', though I may not have good suggestions at the moment. The latter could be something like 'säilö' or 'keskussäilö'. 'pull request' could be 'talletuspyyntö' though that is not an exact word for word translation.